rosidl_generator_dds_idl/cmake/rosidl_generate_dds_interfaces.cmake

```diff
@@ -43,6 +43,12 @@ macro(rosidl_generate_dds_interfaces target)
     if("${_extension}" STREQUAL ".msg")
       get_filename_component(name "${_idl_file}" NAME_WE)
       list(APPEND _generated_files "${_output_path}/${name}_.idl")
+    elseif("${_extension}" STREQUAL ".srv")
+      get_filename_component(name "${_idl_file}" NAME_WE)
```

Comment:
Since this is a macro please prefix the variable with an *underscore*. Since it is used on the if part above too please move the line in front if the conditional.